### PR TITLE
Add Apache Spark 2.3.4 release news and update links

### PR DESCRIPTION
**Pull request by kiszk**
This PR aims tom add the Spark 2.3.4 release news and update links.

### CODE REVIEW

**Comment by kiszk** [comment]
Hi, @srowen , @felixcheung , @dongjoon-hyun, @viirya, @gatorsmile , @cloud-fan, @HyukjinKwon
Could you review this?

File: js/downloads.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ var packagesV8 = [hadoop2p7, hadoop2p6, hadoopFree, sources];
 var packagesV9 = [hadoop2p7, hadoop2p6, hadoopFree, scala2p12_hadoopFree, sources];
 
 addRelease("2.4.4", new Date("08/30/2019"), packagesV9, true);
-addRelease("2.3.3", new Date("02/15/2019"), packagesV8, true);
+addRelease("2.3.4", new Date("09/09/2019"), packagesV8, true);
```
Comment (dongjoon-hyun) [inline js/downloads.js]: Thank you, @kiszk . This should be VOTE pass day which is different from news announce day.

Comment (kiszk) [reply js/downloads.js]: Thank you. I will update the date soon.

Comment (kiszk) [reply js/downloads.js]: @dongjoon-hyun Thank you. Two `downloads.js` have been updated.

**Review by cloud-fan — APPROVED** [review]

File: site/examples.html [hunk 2]
```diff
@@ -230,7 +230,7 @@ <h3>Word Count</h3>
 <div class="tab-pane tab-pane-python active">
 <div class="code code-tab">
 
-<figure class="highlight"><pre><code class="language-python" data-lang="python"><span></span><span class="n">text_file</span> <span class="o">=</span> <span class="n">sc</span><span class="o">.</span><span class="n">textFile</span><span class="p">(</span><span class="s2">&quot;hdfs://...&quot;</span><span class="p">)</span>
```
Comment (viirya) [inline site/examples.html:233]: Does this remove a redundant span?

Comment (kiszk) [reply site/examples.html:233]: I think so. This is automatically done by `jekyll`. Should we revert this?

Comment (viirya) [reply site/examples.html:233]: Not sure. @dongjoon-hyun Do you see such thing when you did the similar change?

Comment (kiszk) [reply site/examples.html:233]: I used v 3.8.6.
```
/usr/local/bin/jekyll -v
jekyll 3.8.6
```

Comment (srowen) [reply site/examples.html:233]: Use the latest jekyll, 4.0.0. I know Hyukjin just updated all the rendering to be consistent with 4.0.0.

Comment (kiszk) [reply site/examples.html:233]: I use v4.0.0. However, the diff looks larger than the current commit.

```
@@ -230,11 +230,11 @@ In this page, we will show examples using RDD API as well as examples using high
 <div class="tab-pane tab-pane-python active">
 <div class="code code-tab">
 
-<figure class="highlight"><pre><code class="language-python" data-lang="python"><span></span><span class="n">text_file</span> <span class="o">=</span> <span class="n">sc</span><span class="o">.</span><span class="n">textFile</span><span class="p">(</span><span class="s2">&quot;hdfs://...&quot;</span><span class="p">)</span>
-<span class="n">counts</span> <span class="o">=</span> <span class="n">text_file</span><span class="o">.</span><span class="n">flatMap</span><span class="p">(</span><span class="k">lambda</span> <span class="n">line</span><span class="p">:</span> <span class="n">line</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s2">&quot; &quot;</span><span class="p">))</span> \
-             <span class="o">.</span><span class="n">map</span><span class="p">(</span><span class="k">lambda</span> <span class="n">word</span><span class="p">:</span> <span class="p">(</span><span class="n">word</span><span class="p">,</span> <span class="mi">1</span><span class="p">))</span> \
+<figure class="highlight"><pre><code class="language-python" data-lang="python"><span class="n">text_file</span> <span class="o">=</span> <span class="n">sc</span><span class="o">.</span><span class="n">textFile</span><span class="p">(</span><span class="s">"hdfs://..."</span><span class="p">)</span>
+<span class="n">counts</span> <span class="o">=</span> <span class="n">text_file</span><span class="o">.</span><span class="n">flatMap</span><span class="p">(</span><span class="k">lambda</span> <span class="n">line</span><span class="p">:</span> <span class="n">line</span><span class="o">.</span><span class="n">split</span><span class="p">(</span><span class="s">" "</span><span class="p">))</span> \
+             <span class="o">.</span><span class="nb">map</span><span class="p">(</span><span class="k">lambda</span> <span class="n">word</span><span class="p">:</span> <span class="p">(</span><span class="n">word</span><span class="p">,</span> <span class="mi">1</span><span class="p">))</span> \
              <span class="o">.</span><span class="n">reduceByKey</span><span class="p">(</span><span class="k">lambda</span> <span class="n">a</span><span class="p">,</span> <span class="n">b</span><span class="p">:</span> <span class="n">a</span> <span class="o">+</span> <span class="n">b</span><span class="p">)</span>
-<span class="n">counts</span><span class="o">.</span><span class="n">saveAsTextFile</span><span class="p">(</span><span class="s2">&quot;hdfs://...&quot;</span><span class="p">)</span></code></pre></figure>
+<span class="n">counts</span><span class="o">.</span><span class="n">saveAsTextFile</span><span class="p">(</span><span class="s">"hdfs://..."</span><span class="p">)</span></code></pre></figure>
```

File: site/sitemap.xml [hunk 3]
```diff
@@ -139,713 +139,721 @@
 </url>
 <!-- Auto-generate sitemap for rest of site content -->
 <url>
```
Comment (srowen) [inline site/sitemap.xml:141]: Oops, did the sitemap get published last time with the localhost:4000 URLs? that's an error. Good change! this one is easy to miss and GIthub collapses this big diff.

File: site/examples.html (hunk 2 above)
Comment (srowen) [reply site/examples.html:233]: If you're using 4.0.0, this should be fine. The difference looks minor.

File: site/sitemap.xml (hunk 3 above)
Comment (kiszk) [reply site/sitemap.xml:141]: Yeah, I realized it. It was introduced unintentionally in the last one.

File: site/examples.html (hunk 2 above)
Comment (kiszk) [reply site/examples.html:233]: Sure, committed the change with jekyll v4.0.0

Comment (viirya) [reply site/examples.html:233]: Interesting. Maybe last change wasn't done with jekyll v4.0.0.

File: site/sitemap.xml (hunk 3 above)
Comment (dongjoon-hyun) [reply site/sitemap.xml:141]: Oh, thanks! This seems to be due to my mistake during `jekyll watch`.

File: site/examples.html (hunk 2 above)
Comment (dongjoon-hyun) [reply site/examples.html:233]: Yes. Mine was 3.8.5 or 3.8.6, not v4.0.0 (IIRC, the version on my laptop at home).
- https://github.com/apache/spark-website/commit/4f850d15a68650384e4c1dd8b74c585ffedc875a (Fix jekyll build before updating)
- https://github.com/apache/spark-website/commit/950c65da1e91419b4e4830a0a23dbc6cb732ddaf (Release v2.4.4)

File: site/examples.html [hunk 4]
```diff
@@ -230,36 +230,36 @@ <h3>Word Count</h3>
...
+             <span class="o">.</span><span class="nb">map</span><span class="p">(</span><span class="k">lambda</span> <span class="n">word</span><span class="p">:</span> <span class="p">(</span><span class="n">word</span><span class="p">,</span> <span class="mi">1</span><span class="p">))</span> \
              <span class="o">.</span><span class="n">reduceByKey</span><span class="p">(</span><span class="k">lambda</span> <span class="n">a</span><span class="p">,</span> <span class="n">b</span><span class="p">:</span> <span class="n">a</span> <span class="o">+</span> <span class="n">b</span><span class="p">)</span>
-<span class="n">counts</span><span class="o">.</span><span class="n">saveAsTextFile</span><span class="p">(</span><span class="s2">&quot;hdfs://...&quot;</span><span class="p">)</span></code></pre></figure>
+<span class="n">counts</span><span class="o">.</span><span class="n">saveAsTextFile</span><span class="p">(</span><span class="s">"hdfs://..."</span><span class="p">)</span></code></pre></figure>
```
Comment (dongjoon-hyun) [inline site/examples.html:237]: Oh, now Jekyll support `<span class="s">"hdfs://..."</span>` instead of `<span class="s2">&quot;hdfs://...&quot;</span>`?

**Comment by dongjoon-hyun** [comment]
~Ur, @cloud-fan . It seems that we still don't have 2.3.4 at Archive directory.~ Never mind. It seems due to a delay.
- https://archive.apache.org/dist/spark/

**Review by dongjoon-hyun — APPROVED** [review]
+1, LGTM.

**Comment by dongjoon-hyun** [comment]
Thank you for updating all here. Later, we can update `README.md` in order to use `v4.0.0` always. `Jekyll v4.0.0` is released 20 days ago. `asf-site` branch seems to be not a result of `v4.0.0` according to the git log.

**Review by viirya — APPROVED** [review]

**Comment by dongjoon-hyun** [comment]
BTW, @cloud-fan and @kiszk . We need to sync `KEYS`, too~
- https://dist.apache.org/repos/dist/dev/spark/KEYS (`Kazuaki Ishizaki (CODE SIGNING KEY)`)
- https://dist.apache.org/repos/dist/release/spark/KEYS (No `Kazuaki Ishizaki (CODE SIGNING KEY)`)

Otherwise,[ the checker](https://checker.apache.org/projs/spark.html) will complain later.

**Review by HyukjinKwon — APPROVED** [review]

**Comment by kiszk** [comment]
@dongjoon-hyun thanks. Let me wait for an additional hour.
When KEY is synced, I will merge this.

**Comment by cloud-fan** [comment]
I've synced the keys

**Comment by kiszk** [comment]
Got it at https://dist.apache.org/repos/dist/release/spark/KEYS!

**Comment by kiszk** [comment]
Thanks, @cloud-fan, @srowen, @dongjoon-hyun, @viirya, and @HyukjinKwon

**Pull request closed without merging**